src/common/hooks/useGeolocationRegions.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
```

Comment:
do you want to add a comment at the top here summarizing the use case/return value of this hook